src/framework/components/light/component.js

```diff
@@ -375,6 +377,9 @@ function _defineProps() {
     _defineProperty('shape', LIGHTSHAPE_PUNCTUAL, function (newValue, oldValue) {
         this.light.shape = newValue;
     });
+    _defineProperty('affectSpecularity', true, function (newValue, oldValue) {
```

Comment:
could you please add this new property to lightComponentProps in types-fixup as well

Reply:
Done 👍